drivers/metal/rendering_context_driver_metal.mm

```diff
@@ -172,7 +172,11 @@ void present(MDCommandBuffer *p_cmd_buffer) override final {
 		count--;
 		front = (front + 1) % frame_buffers.size();
 
-		[p_cmd_buffer->get_command_buffer() presentDrawable:drawable afterMinimumDuration:present_minimum_duration];
+		if (vsync_mode != DisplayServer::VSYNC_DISABLED) {
```

Comment:
Shouldn't this branch still run if `present_minimum_duration` is greater than 0?

This seems to make it so we don't benefit from https://github.com/godotengine/godot/pull/99041 when vsync is disabled. But you are very likely to be using `max_fps` when vsync is disabled.

Reply:
I see, so you are saying when vsync is disabled, but `max_fps` is set, we still expect the renderer to limit the FPS?

Reply:
That's right. We actually handle that in `OS::add_frame_delay` on the CPU side. But using `afterMinimumDuration` allows us to smooth it out and time things with the screen refresh

Reply:
I might have just answered my own question. This probably provides no benefit when not using Vsync if the whole purpose is allowing us to synchronize better with vsync.

Reply:
> Enabling that branch will implicitly enable vsync since macOS will limit frame rate to something the display can support.
> 
> Alternatively, you could query the refresh rate of the display and enable that branch only if the max FPS can be evenly divided by the refresh rate.

Thanks for your response. That makes sense. I agree we should just leave it off when not using vsync.